### PR TITLE
🐛 fix: fix date format in sensors last record. resolve #55

### LINKED ISSUE #55: Last sensors record page data
Get the last sensors records from database and return to the view responsible to render the last sensors record template.

### PR DESCRIPTION
**Pull request by kaufon**

### CODE REVIEW

File: src/app/infra/views/sensors_records_views/last_sensors_record_page_view.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 from infra.repositories import SensorRecordsRepository
 
+
+
 sensors_records_repo = SensorRecordsRepository()
```
Comment (JohnPetros) [inline src/app/infra/views/sensors_records_views/last_sensors_record_page_view.py:7]: sensors_records_repository already exitst in __init__.py of repositories folder. Just import it and use it in the view.

**Review by JohnPetros — CHANGES_REQUESTED** [review]

File: src/app/infra/views/sensors_records_views/last_sensors_record_page_view.py [hunk 2]
```diff
@@ -10,7 +12,7 @@ def last_sensors_record_page_view():
 
     if last_data:
```
Comment (JohnPetros) [inline src/app/infra/views/sensors_records_views/last_sensors_record_page_view.py:8]: If there is no the last record, then the last record should be:

```python
{
"temperature": 0,
"water_volume": 0,
"soil_humidity": 0,
"ambient_humidity": 0,
}

```

**Review by JohnPetros — CHANGES_REQUESTED** [review]

File: src/app/infra/views/sensors_records_views/last_sensors_record_page_view.py (hunk 2 above)
Comment (kaufon) [reply src/app/infra/views/sensors_records_views/last_sensors_record_page_view.py:8]: What about the date

**Review by 0thigs — APPROVED** [review]

File: src/app/infra/repositories/sensors_records_repository.py [hunk 3]
```diff
@@ -61,6 +61,15 @@ def get_last_record(self):
 
             if row:  
                 return self.__get_sensors_record(row)
+            
+            else:
```
Comment (JohnPetros) [inline src/app/infra/repositories/sensors_records_repository.py:65]: Create a use case called get_last_sensors_record, then you put the verification and the exception that you did in the sensors_records_repository inside the use case, then call the use case inside the view:

```python

from core.use_cases.sensors_records import get_last_sensors_record

def last_sensors_record_page_view():
    last_sensors_record = get_last_sensors_record.execute()

    render_template("template_path", last_record=last_sensors_record)

```

Obs: The sensors_records_repository should return None if there is no record.

**Review by JohnPetros — CHANGES_REQUESTED** [review]

File: src/app/infra/repositories/sensors_records_repository.py (hunk 3 above)
Comment (kaufon) [reply src/app/infra/repositories/sensors_records_repository.py:65]: Why add more complexity to it,since its already working in a simple manner?

Comment (JohnPetros) [reply src/app/infra/repositories/sensors_records_repository.py]: This will help a lot to make tests. Trust in me

Comment (kaufon) [reply src/app/infra/repositories/sensors_records_repository.py]: Sure mate!,just wondering to help me understand your plan and improve

**Review by JohnPetros — APPROVED** [review]
Do not forget to do Datetime common to remove the datetime formatting from the view because this belongs to the business rule (core folder).

**Pull request merged**